File: src/main/java/arcana/common/entities/ModEntities.java
package arcana.common.entities;

import arcana.utils.wrappers.ETWrapper;
import net.minecraft.entity.EntityClassification;
import net.minecraft.entity.EntityType;


public class ModEntities {

    public static final EntityType<ArcanumEntity> ARCANUM = new ETWrapper<ArcanumEntity>(ArcanumEntity::new, EntityClassification.MISC, "arcanum")
        .sized(1F, 0.4F).clientTrackingRange(6).updateInterval(20).get();
}

File: src/main/kotlin/arcana/common/entities/ModEntities.kt
package arcana.common.entities

import arcana.utils.wrappers.ETWrapper
import net.minecraft.entity.EntityClassification.MISC

object ModEntities {
    @JvmField val ARCANUM = ETWrapper(MISC, "arcanum", ::ArcanumEntity)
        .sized(1f, 0.4f).clientTrackingRange(6).updateInterval(20).get()
    @JvmField val CHASING_SKULL = ETWrapper(MISC, "chasing_skull", ::ChasingSkullEntity)
        .sized(0.3125F, 0.3125F).clientTrackingRange(4).get()
    @JvmField val DAMAGING_SHOVEL = ETWrapper(MISC, "damaging_shovel", ::DamagingShovelEntity)
        .sized(0.5f, 0.5f).get()
}